modules/apps/portal-language/portal-language-override-service/src/main/java/com/liferay/portal/language/override/internal/PLOLanguageOverrideProvider.java

```diff
@@ -164,19 +87,10 @@ private Map<String, String> _getOverrideMap(
 		return overrideMap;
 	}
 
-	@Reference
-	private CompanyLocalService _companyLocalService;
-
 	@Reference
 	private Language _language;
 
 	private final DCLSingleton<Map<String, HashMap<String, String>>>
 		_ploEntriesMapDCLSingleton = new DCLSingleton<>();
```

Comment:
Hey @regiCesar21, Should it be moved to `PLOLanguageOverrideProviderUtil` as well?